openhands/resolver/examples/openhands-resolver.yml

```
@@ -26,6 +26,7 @@ jobs:
       base_container_image: ${{ vars.OPENHANDS_BASE_CONTAINER_IMAGE || '' }}
       LLM_MODEL: ${{ vars.LLM_MODEL || 'anthropic/claude-3-5-sonnet-20241022' }}
       target_branch: ${{ vars.TARGET_BRANCH || 'main' }}
+      runner: ${{ vars.TARGET_RUNNER || '' }}
```

```suggestion
      runner: ${{ vars.TARGET_RUNNER}}
```

Will specifying a default here override the default `ubuntu-latest` in the workflow definition?

---

No, it doesnt ... already tested, but I will accept your suggestion.